chester/catalog-applications/chester-range.md

```diff
@@ -0,0 +1,241 @@
+---
+slug: chester-range
+title: CHESTER Range
+---
+import Image from '@theme/IdealImage';
+
+# CHESTER Range
+
+This article describes the core functionality, hardware description, and
+example **JSON** message of the catalog application **CHESTER Range**.
+
+:::caution
+
+Some of the basics are not provided, as they are common for all **CHESTER**
+catalog applications. For example, see the article [**Platform
+Management**](../category/platform-management) on how to work with the
+interactive console.
+
+:::
+
+## Application Overview
+
+**CHESTER Range** measures distance using the [MaxBotix
+MB7066](https://shop.hardwario.com/ultrasonic-sensor/) ultrasonic sensor. It
+can also measure temperature using the DS18B20 one-wire sensor and humidity
+using CHESTER S2.
+
+## Application Variants
+
+**CHESTER Range** can be ordered in one of these variants:
+
+### CHESTER Range
+
+The catalog **CHESTER Range** hardware consists of the following ordering
```

Comment:
We will need to include the enclosure part number. Discuss with Robert + Štěpán to create this enclosure.